packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -299,7 +299,8 @@ const ruleset: any = {
       message: "{{description}}",
       disableForTypeSpec: true,
       disableForTypeSpecReason: "Covered by TSP's '@azure-tools/typespec-azure-resource-manager/no-record' rule.",
-      resolved: true,
+      // Only report errors at the source, not from inside $refs (the resolved document)
+      resolved: false,
```

Comment:
FYI I believe the `resolved` is defined here:
https://docs.stoplight.io/docs/spectral/d3482ff0ccae9-rules#resolved

Reply:
Thanks.  After reading that, I feel even more confident this rule should use `resolved: false`.